packages/test-support/addon/jsonapi-factory.js

```diff
@@ -25,39 +42,49 @@ export default class JSONAPIFactory {
     return new ResourceFactory(resource);
   }
 
-  getModels() {
+
+  getModelsFor(type, id) {
```

Comment:
The name of this method confused me. But also, it returns the wrong format (based on where you're calling this, as the return value from schema.js files). A schema.js file should not return an array. It should return a complete json:api document, with the single card definition as `data` and all the related stuff in `included`.

Reply:
got it 👍